### DISCUSSION TITLE
logging with multiple workers

### DISCUSSION THREAD

**Discussion by gygyor** [fastapi/fastapi#4634] [Questions]
What is the standard way of initializing logging in a multi-worker situation? 
I have inited standard python console and file logger in **api.py** and I am running uvicorn from command-line. 
This causes problems when multiple workers are present.

**Comment by zentixua** [comment]
same

**Reply by Kludex** [reply]
What's the issue? Do you have an MRE?

**Reply by zentixua** [reply]
1. The longer the web server is running, the more requests are not logged.  Over time, logging works "at times" (at intervals of 2-3 minutes), and then the web server is fully freezing 🥶
2. 

This only happens when logging to a file
